spec/Action/CaptureActionSpec.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * another great project.
+ * You can find more information about us on https://bitbag.shop and write us
+ * an email on mikolaj.krol@bitbag.pl.
+ */
```

Comment:
Please, change your references to ours:
You can use contact@payplug.com

Reply:
This was part of our template header we add to plugins under @BitBagCommerce organization. As it's not there, I removed it completely.